instrumental/drivers/motion/_kinesis/isc.py

```diff
@@ -73,22 +75,34 @@ def _start_polling(self, polling_period='200ms'):
         self.polling_period = polling_period
         self.dev.StartPolling(self.polling_period.m_as('ms'))
 
+    def get_info(self):
+        description = NiceISC.GetDeviceInfo(self.serial).description
+        return ffi.string(description)
+
     @check_units(angle='deg')
-    def move_to(self, angle, wait=False):
-        """Rotate the stage to the given angle
+    def move_to(self, angle, wait=False, move_type='abs'):
+        """Rotate the stage to the given angle in absolute or relative units
 
         Parameters
         ----------
         angle : Quantity
             Angle that the stage will rotate to. Takes the stage offset into account.
+        wait: (bool) if True waits to receive message telling the move is done else returns
+        move_type: (str) either 'abs' (default) for absolute positioning or 'rel' for relative positioning
         """
         log.debug("Moving stage to {}".format(angle))
         log.debug("Current position is {}".format(self.position))
         self.dev.ClearMessageQueue()
-        self.dev.MoveToPosition(self._to_dev_units(angle + self.offset))
+        if move_type == 'abs':
+            self.dev.MoveToPosition(self._to_dev_units(angle + self.offset))
+        else:
+            self.dev.MoveRelative(self._to_dev_units(angle))
```

Comment:
I thought I posted this before, but maybe I accidentally deleted it. Can you instead create a new `move_relative` method (or perhaps `move_rel` if you prefer)?

Reply:
ok created the move_relative method. On that subject, I would like to create  a common motion interface in the motion object. If you are ok, I would like to introduce this move_to and move_relative in it? I'll make a separated PR for this anyway